doc/refman/RefMan-ltac.tex

```diff
@@ -156,6 +156,8 @@ \section{Syntax}
 & | & {\tt type\_term} {\term}\\
 & | & {\tt numgoals} \\
 & | & {\tt guard} {\it test}\\
+& | & {\tt assert\_fails} {\tacexprpref}\\
+& | & {\tt assert\_succeds} {\tacexprpref}\\
```

Comment:
Is this really the right place to mention combinators that are not part of the base Ltac grammar? Moreover, adding it to the grammar seems a bit weak as a documentation. Don't the other entries have more complete descriptions?